plugin/helpers_bots.go

```diff
@@ -57,6 +129,10 @@ func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error)
 		} else {
 			p.API.LogError("Plugin attempted to use an account that already exists. Convert user to a bot account in the CLI by running 'mattermost user convert <username> --bot'. If the user is an existing user account you want to preserve, change its username and restart the Mattermost server, after which the plugin will create a bot account with that name. For more information about bot accounts, see https://mattermost.com/pl/default-bot-accounts", "username", bot.Username, "user_id", user.Id)
 		}
+		setImagesErr := p.setBotIconAndProfileImage(user.Id, o.ProfileImagePath, o.IconImagePath)
+		if setImagesErr != nil {
+			return user.Id, setImagesErr
+		}
 		return user.Id, nil
 	}
 
```

Comment:
i believe we missed the case where we created the bot the first time and we want to update the profile/icon paths, no?

Reply:
@ali-farooq0 Yes you're right. I'll make the change.

Reply:
Thanks @alxsah!